tss-esapi/src/context/tpm_commands/attestation_commands.rs

```diff
@@ -12,7 +12,45 @@ use std::convert::TryFrom;
 use std::ptr::null_mut;
 
 impl Context {
-    // Missing function: Certify
+    /// Prove that an object is loaded in the TPM
+    ///
+    /// # Errors
```

Comment:
Sorry to be "that guy" but could you also add more thorough command description? Something like this: https://github.com/parallaxsecond/rust-tss-esapi/pull/248/files#diff-c58cba8761b8466b464d02417779fc58f2cff4200e47d5937b3a1b9de6890bfaR18-R294

The code example can be similar to the test case that you already have.

Reply:
Good suggestion. I'll try to add something as good as the example you link to.